.ci/Jenkinsfile

```diff
@@ -220,6 +220,66 @@ pipeline {
         }
       }
     }
+    stage('Stack Release Tag') {
```

Comment:
I'd say to include this particular stage as a nested stage in `stage('Release') {`

Reply:
I moved the whole logic within the release stage.